src/classes/issues-processor.ts

```diff
@@ -781,6 +796,26 @@ export class IssuesProcessor {
     return this.options.anyOfLabels;
   }
 
+  private _shouldRemoveStaleWhenUpdated(issue: Issue): boolean {
+    if (issue.isPullRequest) {
```

Comment:
I wonder if we could maybe consolidate this function? There seem to be a lot of `if/else` when we might just be able to return the boolean value the option e.g. `removePrStaleWhenUpdated`. At a minimum, I think it could just be an `else` and no need for the `else if` to check the opposite condition

Reply:
I removed two else if, nice catch